src/lib/nodes/abstract.node.ts

```diff
@@ -180,7 +180,8 @@ export abstract class Node<C = unknown> {
       height: this.height,
       status: this.getNodeStatus(),
       type: this.type,
-      label: this.label
+      label: this.label,
+      formatHeight: this.formatHeight(this.height)
```

Comment:
Let's rename the property to `formattedHeight` to not confuse with the `formatHeight` method.